<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenameSandboxDialog</class>
 <widget class="QWidget" name="RenameSandboxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>150</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Rename Sandbox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0" colspan="3">
      <widget class="QLineEdit" name="txtBoxAlias"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblNamePrompt">
       <property name="text">
        <string>Please enter a new name for the Sandbox.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QLabel" name="lblAliasPrompt">
       <property name="text">
        <string>Please enter a new alias for the Sandbox.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLineEdit" name="txtBoxName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QCheckBox" name="chkAliasDisabled">
       <property name="text">
        <string>Disable alias</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkHideAlias">
       <property name="text">
        <string>Hide alias</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerButtons">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Policy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtBoxName</tabstop>
  <tabstop>chkAliasDisabled</tabstop>
  <tabstop>chkHideAlias</tabstop>
  <tabstop>txtBoxAlias</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
